docs/security/authorization/spark/install.md

```diff
@@ -74,6 +74,25 @@ for pointing to the right Ranger admin server.
 
 </configuration>
 ```
+- Additional configs for using Marcos in row-level filter
```

Comment:
##### h5. Using Marcos in Row Level Filters

Reply:
alright.